cyclic-position-benchmark/runcycletestsandplot.py

```diff
@@ -0,0 +1,582 @@
+import collections
+import json
+import math
+import os
+import subprocess
+import time
+from threading import Thread
+import sgfmill
+import sgfmill.sgf
+import sgfmill.boards
+import sgfmill.ascii_boards
+import matplotlib.pyplot as plt
+import numpy as np
+import seaborn as sns
+
+PLOTS_PATH = "generated_plots"  # path to output plots
+SGFS_PATH = "sgfs"  # path to input positions
+KATAGO_PATH = "/engines/KataGo-raw/cpp/katago"  # path to compiled KataGo
+MODELS_PATH = "/go_attack/victim_models/"  # path to models to be tested
```

Comment:
Could use `argparse` instead to make these changeable via command line instead of needing the user to modify the code

Reply:
sure, done